ironfish-rust/src/keys/mod.rs

```diff
@@ -29,7 +29,7 @@ pub use view_keys::*;
 mod test;
 
 const EXPANDED_SPEND_BLAKE2_KEY: &[u8; 16] = b"Iron Fish Money ";
-
+pub const KEY_SIZE: usize = 32;
```

Comment:
`SPEND_KEY_SIZE` or something might be a little more clear - we deal with a lot of different keys

Reply:
Aren't all the keys the same size?

Reply:
Note: yes they will most likely stay the same size